src/vault.py

```diff
@@ -39,6 +39,24 @@ def check_ip(ip: str):
         sys.exit(1)
 
 
+def check_ip_range(ip_range: int):
+    """
+    Check whether the input IP range is valid or not
+    """
+    try:
+        number = int(ip_range)
```

Comment:
It would be better if we use a suitable variable name instead of `number`.

Reply:
Any suggestion?

Reply:
Why not `ip_range` ?

Reply:
I think that this problem is solved now